### PR TITLE
Add release documentation for `-XX:[+|-]ShowHiddenFrames`

### PR DESCRIPTION
**Pull request by fengxue-IS**
Related: #3627 

Signed-off-by: Jack Lu <Jack.S.Lu@ibm.com>

### CODE REVIEW

**Comment by fengxue-IS** [comment]
@pshipton can you please take a look

File: doc/release-notes/0.31/0.31.md [hunk 1]
```diff
@@ -48,6 +48,13 @@ The following table covers notable changes in v0.31.0. Further information about
 </thead>
 <tbody>
 
+<tr>
+<td valign="top"><a href="https://github.com/eclipse-openj9/openj9/pull/3627">#3267</a></td>
+<td valign="top">New <tt>-XX:[+|-]ShowHiddenFrames</tt> option added</td>
+<td valign="top">All versions</td>
+<td valign="top">This option controls if Hidden method frames JVM generated for MethodHandles are displayed in stacktrace. This option is disabled by default and can be enabled for debugging purposes.</td>
```
Comment (pshipton) [inline doc/release-notes/0.31/0.31.md:55]: `Hidden` doesn't need to be capitalized.
You can remove `JVM`.
`stacktrace` should be `a stacktrace`.

**Review by pshipton — APPROVED** [review]

**Comment by fengxue-IS** [comment]
@pshipton should I create a PR for 0.31 branch?

**Comment by pshipton** [comment]
Sure. We don't necessarily keep the release notes in the branch up to date, but we have lots of time to update this one.

**Pull request merged**